testing/xfpga/test_metrics_utils_c.cpp

```diff
@@ -671,9 +641,7 @@ TEST_P(metics_uitls_dcp_c_p, test_metric_utils_12) {
 
 
 	EXPECT_EQ(FPGA_OK, fpga_vector_free(&vector));
-
-	dlclose(_handle->bmc_handle);
-
+	//dlclose(_handle->bmc_handle);
```

Comment:
Remove if it isn't needed.

Reply:
@anandaravuri Is it okay to remove this?